kronolith/lib/Ajax/Application/Handler.php

```diff
@@ -352,10 +352,15 @@ public function saveEvent()
         }
 
         if (($result !== true) && $this->vars->sendupdates) {
-            $type = $event->status == Kronolith::STATUS_CANCELLED
-                ? Kronolith::ITIP_CANCEL
-                : Kronolith::ITIP_REQUEST;
-            Kronolith::sendITipNotifications($event, $notification, $type);
+            if ($this->vars->attendance) {
+                Kronolith::sendITipNotifications($event, $notification, Kronolith::ITIP_REPLY);
+            }
+            if (!$event->organizer || Kronolith::isUserEmail($this->creator, $event->organizer)) {
+                $type = $event->status == Kronolith::STATUS_CANCELLED
+                    ? Kronolith::ITIP_CANCEL
+                    : Kronolith::ITIP_REQUEST;
+                Kronolith::sendITipNotifications($event, $notification, $type);
+            }
```

Comment:
I had a look at the Javascript code for that but as far as I could see we don't have the event data available where the Dialog is called / made visible.
But I must admit that my Javascript knowledge isn't the best, so if you'd like to have a look at that I'd be thankful.